tests/test_fuzz.py

```diff
@@ -77,7 +77,15 @@
 
 
 def onlyIfHasHypothesis(has_hypothesis):
-	"""Decorator to handle optional loading."""
+	"""Conditionally enable a class based on the availability of the hypothesis library.
```

Comment:
@coderabbitai, (NIT) - these changes are not aligned with [CEP-7](https://gist.github.com/reactive-firewall/123b8a45f1bdeb064079e0524a29ec20); namely:
> While PEP-8 and PEP-257 allow single-line docstrings for simple functions, CEP-7 PREFERS multi-line docstrings to maintain consistency.

```suggestion
	"""
	Conditionally enable a class based on the availability of the hypothesis library.
```